libraries/AP_Parachute/AP_Parachute.h

```diff
@@ -104,6 +108,7 @@ class AP_Parachute {
     AP_Int16    _servo_off_pwm; // PWM value to move servo to when shutter is deactivated
     AP_Int16    _alt_min;       // min altitude the vehicle should have before parachute is released
     AP_Int16    _delay_ms;      // delay before chute release for motors to stop
+    AP_Int16     _timeout;      // loss of control timeout value in seconds
```

Comment:
If its in seconds I think it should be a float.

Reply:
I have corrected it. What do you think about the range from 0.5 to 5 seconds?